src/ffi/mod.rs

```diff
@@ -1301,6 +1301,30 @@ pub extern "C" fn telio_notify_network_change(
     })
 }
 
+#[no_mangle]
```

Comment:
Do not forget to Update `src/libtelio.udl` as well ;). 

Note that at least theoretically one should also remove it from swig bindings, but those are going to be removed soon, so probably no point in updating them anymore, as `main` is now on the track for v5.0.0 release, which will be UniFFI only.

Reply:
Done!